arch/arm/boot/dts/overlays/README

```diff
@@ -807,6 +807,7 @@ Params: cam0-arducam-64mp       Select Arducam64MP for camera on port 0
         cam0-imx708             Select IMX708 for camera on port 0
         cam0-ov2311             Select OV2311 for camera on port 0
         cam0-ov5647             Select OV5647 for camera on port 0
+        cam0-ov6a40             Select OV64A40 for camera on port 0
```

Comment:
These added parameters are all misnamed, but the overlay looks good once they've been renamed.

Reply:
oh that was really stupid, sorry, I didn't notice. Fixed a re-pushed